### PR TITLE
Add solver params

### PR DESCRIPTION
**Pull request by nico-canta**
Within this PR, the new API changes the arguments of this function:

```python
update_static_flow_pressure(graph, boundary_flow, params)
```

### CODE REVIEW

File: astrovascpy/bloodflow.py [hunk 1]
```diff
@@ -632,12 +639,13 @@ def construct_static_incidence_matrix(graph):
     return sparse.csc_matrix((data, (row, col)), shape=(graph.n_edges, graph.n_nodes))
 
 
-def _solve_linear(laplacian, input_flow):
+def _solve_linear(laplacian, input_flow, params=None):
```
Comment (cattabiani) [inline astrovascpy/bloodflow.py:642]:
```suggestion
def _solve_linear(laplacian, input_flow, params={}):
```
This is in conjunction with the suggestion later

File: astrovascpy/bloodflow.py [hunk 2]
```diff
@@ -654,6 +662,13 @@ def _solve_linear(laplacian, input_flow):
     """
     )
 
+    if params is None:
+        params = {}
```
Comment (cattabiani) [inline astrovascpy/bloodflow.py:666]:
```suggestion
```

**Review by cattabiani — APPROVED** [review]
One knitpick but not blocking. GJ!

File: astrovascpy/bloodflow.py (hunk 1 above)
Comment (matz-e) [reply astrovascpy/bloodflow.py:642]: No mutable data structures as Python default arguments? I would not do that!

Comment (cattabiani) [reply astrovascpy/bloodflow.py:642]: the changes to params does not need to affect the outside. In this case it is totally safe

**Pull request merged**